py2-requirements.txt

```diff
@@ -1,23 +1,24 @@
-autopep8==1.3.1
+autopep8==1.3.4
 backoff==1.5.0
-boto==2.46.1
-botocore==1.5.95
-boto3==1.4.4
-coverage==4.3.4
+boto==2.48.0
+boto3==1.6.8
+botocore==1.9.8
+coverage==4.5.1
 Fabric==1.13.2
-green==2.7.0
-kids.cache==0.0.4
+green==2.12.1
+kids.cache==0.0.7
 mock==2.0.0
 more-itertools==4.1.0
 netaddr==0.7.19
-pyflakes==1.5.0
+pyflakes==1.6.0
 pylint==1.8.2
 pytest==3.5.0
-python-dateutil==2.6.0
-python-slugify==1.2.1
+# botocore 1.9.8 has requirement python-dateutil<2.7.0,>=2.1
+python-dateutil==2.6.1
+python-slugify==1.2.4
 git+https://github.com/beelit94/python-terraform.git@develop#egg=python-terraform
-pytz==2016.10
+pytz==2018.3
 PyYAML==3.12
-requests==2.13.0
-troposphere==1.9.2
+requests==2.18.4
+troposphere==2.2.1
```

Comment:
new major version :thinking: 
Where did all this come from? Should not be tied to the update of Salt

Reply:
It seems it is bringing Python 2 dependencies up to the same versions as Python 3 one, but it should be a separate PR; perhaps there is some bug that needed fixing via upgrading?